src/main/scala/org/opennetworkinsight/utilities/DomainProcessor.scala

```diff
@@ -2,12 +2,12 @@ package org.opennetworkinsight.utilities
 
```

Comment:
hmmm

maybe we should just pull in a third-party open source library for this stuff

https://github.com/NET-A-PORTER/scala-uri

is Apache 2 licensed

but who knows how long it will be supported, etc.